troubleshooting/index.adoc

```diff
@@ -0,0 +1,12 @@
+= Troubleshooting
+Marián Hlaváč <marian.hlavac@fit.cvut.cz>
```

Comment:
do we generally attribute individual authors?

Reply:
let's not, it's a mix. we can add attribution page where we list all authors

Reply:
Nice thing, when defined author this way, is that on courses.fit.cvut.cz the author is displayed in the footer, even when someone else made the last commit. I think it should be used for someone who did most of the writing on that page. But the choice is yours ofc.